target/linux/ath79/image/common-compex.mk

```diff
@@ -0,0 +1,7 @@
+define Device/compex_wpj-16m
+# TODO wpj563 usb3
+  ATH_SOC := qca9531
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
+  MTDPARTS := spi0.0:192k(u-boot)ro,16128k(firmware),64k(art)ro
```

Comment:
This is obsolete, as you define partitions in DTS.

Reply:
thanks :-)